domain/src/main/java/edu/nextstep/camp/calculator/domain/Expression.kt

```diff
@@ -0,0 +1,70 @@
+package edu.nextstep.camp.calculator.domain
+
+
+class Expression {
+    var expressionText = ""
```

Comment:
```suggestion
    var expressionText = ""
        private set
```
expressionText는 외부에서 변경이 가능해요.
내부에서만 변경이 가능하도록 코드를 변경해보면 어떨까요?